app/templates/owner/getWorkersView.html

```diff
@@ -0,0 +1,25 @@
+{% extends "owner/layout.html" %}
+{% block title %}Workers{% endblock %}
+{% block head %}
+    {{ super() }}
+    <style>
+    </style>
+{% endblock %}
+{% block body %}
+    <h1>Workers</h1>
+    <table class=users>
+        <tr>
+            <th>#
```

Comment:
Ja nie wiem czy wam jakieś IDE to tak poprawia ale niedomykanie tagów to jest barbarzyństwo. Jak chcecie to inaczej robić i nie bawić się html to poczytajcie sobie o haml.